extension/src/background/helpers/account.ts

```diff
@@ -127,6 +132,28 @@ export const getNetworksList = async () => {
   return networksList;
 };
 
+export const getIsRpcHealthy = async (networkDetails: NetworkDetails) => {
+  let rpcHealth = { status: "" };
+  if (isCustomNetwork(networkDetails)) {
+    // TODO: use server.getHealth method to get accurate result for standalone network
+    rpcHealth = { status: "healthy" };
```

Comment:
weirdly, I was having trouble using the sdk method for checking rpc health for my local Standalone. I'll take another look at this, but I think Mainnet/Testnet is the priority here